pootle/apps/pootle_store/forms.py

```diff
@@ -279,16 +283,42 @@ def clean_target_f(self):
 
             return value
 
-        def clean_state(self):
+        def clean_similarity(self):
```

Comment:
As I see the `clean_similarity()` and `clean_mt_similarity()` methods have been moved with no change/intent. If there was a cleanup intention please consider doing so separately.